src/backend/expungeservice/expunger.py

```diff
@@ -194,7 +199,7 @@ def _most_recent_convictions(recent_convictions):
             return newer
 
     @staticmethod
-    def _is_other_convictions_all_traffic(convictions):
+    def _are_other_convictions_all_traffic(convictions):
```

Comment:
While using `are` might be grammatically correct, I've always seen `is` used for any boolean return type method name where applicable.

Reply:
Will revert this in the next revision.